src/status.c

```diff
@@ -676,7 +676,7 @@ open_mergetool(const char *file)
 {
 	const char *mergetool_argv[] = { "git", "mergetool", file, NULL };
 
-	open_external_viewer(mergetool_argv, repo.cdup, false, true, false, true, "");
+	open_external_viewer(mergetool_argv, repo.cdup, false, true, false, true, true, "");
```

Comment:
Not sure what the motivation is for using the quick mode here. I'll change it to the original behavior.